traefik_ingress/variables.tf

```diff
@@ -15,18 +15,6 @@ variable "plugin" {
   })
 }
 
-variable "args" {
```

Comment:
Why remove these?

Reply:
As of now it was only used for the removed monitoring module. We can keep it for the sake of futureproofing.